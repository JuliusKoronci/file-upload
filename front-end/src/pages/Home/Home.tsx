import * as React from 'react';
import { Card, CardContent, Grid } from '@material-ui/core';

import { DocumentList, UploadForm } from '../../features';
import { styles } from '../../styles';
import { DocumentsProvider } from '../../providers';

export const Home = () => (
  <div style={styles.layout}>
    <DocumentsProvider>
      <Grid container={true} spacing={24}>
        <Grid item={true} xs={12}>
          <h3>FILE UPLOAD</h3>
        </Grid>
        <Grid item={true} xs={12}>
          <Card>
            <CardContent>
              <Grid item={true} xs={12}>
                <UploadForm />
              </Grid>
              <Grid item={true} xs={12}>
                <DocumentList />
              </Grid>
            </CardContent>
          </Card>
        </Grid>
      </Grid>
    </DocumentsProvider>
  </div>
);