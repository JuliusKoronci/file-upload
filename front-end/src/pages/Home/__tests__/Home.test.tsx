import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Home } from '../Home';


describe('Home', () => {
  it('should render snapshot', () => {
    expect(toJson(shallow(<Home />))).toMatchSnapshot();
  });
});