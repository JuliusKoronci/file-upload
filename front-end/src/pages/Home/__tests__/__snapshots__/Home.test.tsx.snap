// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render snapshot 1`] = `
<div
  style={
    Object {
      "margin": "0 auto",
      "maxWidth": "1224px",
      "minWidth": "768px",
    }
  }
>
  <Connect(DocumentsProvider)>
    <WithStyles(Grid)
      container={true}
      spacing={24}
    >
      <WithStyles(Grid)
        item={true}
        xs={12}
      >
        <h3>
          FILE UPLOAD
        </h3>
      </WithStyles(Grid)>
      <WithStyles(Grid)
        item={true}
        xs={12}
      >
        <WithStyles(Card)>
          <WithStyles(CardContent)>
            <WithStyles(Grid)
              item={true}
              style={
                Object {
                  "textAlign": "right",
                }
              }
              xs={12}
            >
              <Connect(Component) />
            </WithStyles(Grid)>
            <WithStyles(Grid)
              item={true}
              xs={12}
            >
              <withState(Connect(UploadContainer)) />
            </WithStyles(Grid)>
            <WithStyles(Grid)
              item={true}
              xs={12}
            >
              <Connect(Component) />
            </WithStyles(Grid)>
          </WithStyles(CardContent)>
        </WithStyles(Card)>
      </WithStyles(Grid)>
    </WithStyles(Grid)>
  </Connect(DocumentsProvider)>
</div>
`;
